set-mongo/src/main/java/eu/europeana/set/web/model/WebUserSetImpl.java

```diff
@@ -93,6 +93,12 @@ public int getPrev() {
         return super.getSubject();
     }
 
+    @Override
+    @JsonProperty(WebUserSetFields.PINNED)
```

Comment:
but until the value is more than 0, the field will not appear.

Reply:
Only if the following annotation is added. Also, we have to use JsonGetter instead of JsonProperty annotation, as this field must not be updated though the update method

Reply:
if we add the custom filter the pinned value will appear for all the sets as pinned: 0.

Reply:
not if we value is set to -1 for the other sets. This could be implemented directly into the get method

Reply:
done